lemur/certificates/verify.py

```diff
@@ -55,21 +57,33 @@ def ocsp_verify(cert, cert_path, issuer_chain_path):
         stderr=subprocess.PIPE,
     )
 
-    message, err = p2.communicate()
+    try:
+        message, err = p2.communicate(timeout=15)
+    except TimeoutExpired:
+        p2.kill()
+        p2.communicate()
+        return False
```

Comment:
Returning False would mean cert is revoked (not valid), if so I think it is safer to retry in future run and return True in case of timeout.

Reply:
excellent point, will raise exception, consistent with other errors.